components/tikv_util/src/lib.rs

```diff
@@ -487,6 +485,7 @@ pub fn set_panic_hook(panic_abort: bool, data_dir: &str) {
                 // static variables of RocksDB, which may cause other threads encounter
                 // pure virtual method call. So calling libc::_exit() instead to skip the
                 // cleanup process.
+                libc::fflush(std::ptr::null_mut());
```

Comment:
Why? Cache are usually line buffered, no need to flush again.

Reply:
I don't get what you mean. Do it in case the log is not flushed in time, see https://www.ibm.com/docs/en/zos/2.1.0?topic=functions-exit-end-process-bypass-cleanup

Reply:
If logs are written to terminal, they are flush line by line, so no need to call `fflush`; if logs are written to file, content will be written to file immediately at write.

Reply:
Oh, I see. The flush is called explicitly in `log` of `TikvFormat`

Reply:
Seems it's indeed unnecessary. But maybe it's better to do it to make it close to the behavior of `exit()`?

Reply:
> make it close to the behavior of `exit()`

Don't see any point in doing this. Better keep code clean.